docs/data/migration/migration-pickers-v5/migration-pickers-v5.md

```diff
@@ -36,9 +36,62 @@ If you have already applied the `v6.0.0/preset-safe` codemod, then you should no
 
 All other changes must be handled manually.
 
-## Picker components rewrite
+## Picker components
 
-### Update format of the `value` prop
+### New picker components
```

Comment:
This section would be removed when removing the legacy pickers
It is mainly here to help people trying out the beta understand what is going on